README.md

```diff
@@ -34,3 +34,10 @@ val subscriber = kafka.publish("uppercaseStrings", "groupName")
 
 Source(publisher).map(_.toUpperCase).to(Sink(subscriber)).run()
 ```
+
+Tuning
+----
+
+KafkaActorSubscriber and KafkaActorPublisher have their own thread pools, configured in `application.conf`.
```

Comment:
Hey there @amorfis, as a library this repo should *not* contain any `application.conf` files.
Instead it should provide `reference.conf` with "default values" (exactly the contents of your current application.conf).
The reason for this is that users of your library will provide `application.conf` files which *must* take precedence over reference conf - so they are able to override your defaults. Refer to https://github.com/typesafehub/config#longer-examples for more rationale